src/common/buffer.cc

```diff
@@ -1118,6 +1118,12 @@ static simple_spinlock_t buffer_debug_lock = SIMPLE_SPINLOCK_INITIALIZER;
     return true;
   }
 
+  bool buffer::list::can_zero_copy(const char *dst) const
```

Comment:
there's already a can_zero_copy() method for buffer lists dealing with another meaning of zero copy.

Maybe call this one is_buf() or something instead, since it's checking whether the bufferlist only uses the provided buffer?